docs/source/contribute.rst

```diff
@@ -2,12 +2,64 @@
+There are plenty of ways you can contribute to the various parts that make up Vircadia.
 
-Our `GitHub repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds. The project embraces distributed development. If you find a small bug and have a fix, pull requests are welcome. 
+.. contents:: On This Page
+    :depth: 2
 
------------------
-Developer Section
------------------
+-------------
+Documentation
+-------------
+You can help create and maintain the documentation in several ways:
 
-All information required to build from source and develop the project is found in the :doc:`Developer Section <developer>`.
\ No newline at end of file
+- Report errors or missing information to the `GitHub issue tracker <https://github.com/kasenvr/vircadia-docs-sphinx/issues>`_.
+- Create or change documentation pages and open a `PR (pull request) on GitHub <https://github.com/kasenvr/vircadia-docs-sphinx/pulls>`_.
+- `Review changes <https://github.com/kasenvr/vircadia-docs-sphinx/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+review">`_ to the documentation on Github. We would like to have two reviewers for most changes, as there are always things that the writer and the first reviewer overlook.
+
+
+-----------
+Translation
+-----------
+As an international project, Vircadia relies on translators to be able to reach users that do not speak English.
+Currently the only component that has translation support is the documentation.
+You can help translating using our online translation system `Weblate <https://weblate.vircadia.dev>`_ by either submitting translations or becoming a maintainer for your language and checking translations before they make it into the documentation.
+More information on translating the documentation can be found in its `README <https://github.com/kasenvr/vircadia-docs-sphinx/blob/master/README.md>`_.
+
+---------
+Donations
+---------
+Vircadia is primarily driven by the voluntary contributions of its community and core team.
+While we do not have an entity to manage donations, you can donate to specific members of the team via various services like PayPal, Patreon, or Github Sponsors.
+
+You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_.
+Of course, bounties are also an option for other parts of the project like the documentation.
+Bounties are created and assigned based on trust, so if you are unsure about something regarding one then speak to a core developer or bring it up at a community or developer meeting.
+Most of the developers are highly trained and experienced, normally earning upwards of $40 an hour in the free market. Keep that in mind if you want to see something accomplished that no one else is personally interested in.
```

Comment:
```suggestion
Most of the developers are highly trained and experienced, normally earning upwards of US$40 an hour in the free market. Keep that in mind if you want to see something accomplished that no one else is personally interested in.
```

Reply:
One last thing, heh, I realized that $40 may not make sense to everyone unless specified to be USD. :)